alphadia/constants/default.yaml

```diff
@@ -129,6 +129,8 @@ search_output:
   num_samples_quadratic: 50
   min_nonnan: 3
   normalize_lfq: True
+  # can be either "parquet" or "tsv"
+  file_format: "parquet"
```

Comment:
shall we really make parquet the default already? maybe wait until it's tested a bit more?

Reply:
good point